src/js/views/patients/sidebar/filters/filters-sidebar_views.js

```diff
@@ -91,18 +91,35 @@ const FiltersCountView = View.extend({
   },
 });
 
+const ClearFiltersView = View.extend({
+  template: hbs`
+    <button class="filters-sidebar__clear-filters u-margin--r-16 js-clear-filters" {{#if isDisabled}}disabled{{/if}}>
+      {{ @intl.patients.sidebar.filters.filtersSidebarViews.clearFilters }}
```

Comment:
`filtersSidebarViews.clearFiltersView.clearFilters`

Reply:
Since this is now in the `HeaderView`, I moved this to `@intl.patients.sidebar.filters.filtersSidebarViews.headerView.clearFilters`.